.travis.yml

```diff
@@ -28,6 +28,7 @@ addons:
             - g++-4.8
             - cmake
             - python3-dev
+            - libasio-dev
```

Comment:
temporary workaround (before this package is added to the whitelist) suggestion:

in the `install` section:

~~~
- wget 'https://01.org/sites/default/files/asio-1.10.6.tar.gz'
- tar xf asio-1.10.6.tar.gz -C $PREFIX --strip-components=1
~~~